test/end-to-end/paths/intestacy/card.cancellation.en.test.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const taskListContent = require('app/resources/en/translation/tasklist');
+const TestConfigurator = new (require('test/end-to-end/helpers/TestConfigurator'))();
+
+const optionYes = '';
+const optionNo = '-2';
+const ihtOnline = '-2';
+const maritalStatusMarried = '';
+const spouseOfDeceased = '';
+const bilingualGOP = false;
+const uploadingDocuments = false;
+
+Feature('Grant Of Probate Intestacy negative scenario E2E Tests...');
+
+// eslint complains that the Before/After are not used but they are by codeceptjs
+// so we have to tell eslint to not validate these
+// eslint-disable-next-line no-undef
+Before(() => {
```

Comment:
Don' t we have .eslintrc.json where we can declare global like this

https://github.com/hmcts/div-petitioner-frontend/blob/55b1eb675ab1f4d2dd23ad66456ae69ec41dbd57/test/.eslintrc.json

    "globals": {
        "actor": true,
        "Feature": true,
        "Scenario": true,
        "xScenario": true,
        "Before": true,
        "After": true,
        "BeforeSuite": true,
        "AfterSuite": true,
        "codecept_helper": true
    }